code/modules/mob/living/carbon/human/species/station/traits_vr/neutral.dm

```diff
@@ -19,19 +19,19 @@
 	var_changes = list("metabolic_rate" = 1.4, "hunger_factor" = 0.4, "metabolism" = 0.012) // +40% rate and 8x hunger (Double Teshari)
 	excludes = list(/datum/trait/metabolism_up, /datum/trait/metabolism_down)
 
-/datum/trait/cold_discomfort
-	name = "Hot-Blooded"
-	desc = "You are too hot at the standard 20C. 18C is more suitable. Rolling down your jumpsuit or being unclothed helps."
+/datum/trait/coldadapt
+	name = "Cold adapted"
```

Comment:
"Cold-Adapted", or "Hot-Blooded" work here. Sorry to be super strict on grammar/spellcheck, but it's a pet-peeve of mine.